### PR TITLE
Create tutorial/welcome page

### PR DESCRIPTION
**Pull request by Drvanon**
This would resolve issue #18.

### CODE REVIEW

File: gridt/src/app/about/about.page.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+ion-img {
+    max-width: 50vw;
+    max-height: 50vw;
+    overflow: hidden;
+}
+
+p {
+    padding: 1em;
+}
```
Comment (Drvanon) [inline gridt/src/app/about/about.page.scss:15]: It is a good idea to limit addressing tags directly, instead use something like `ion-slides.intro-slides`, or something along those lines to make sure you don't change classes in other parts of the application, as well as keeping your intentions very clear.

File: gridt/src/app/about/tutorial.guard.spec.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [TutorialGuard]
+    });
+  });
+
+  it('should ...', inject([TutorialGuard], (guard: TutorialGuard) => {
+    expect(guard).toBeTruthy();
+  }));
+});
```
Comment (Drvanon) [inline gridt/src/app/about/tutorial.guard.spec.ts:15]: There are issues in all of your tests. Please try to resolve these.

**Comment by Yori-O92** [comment]
There is a test that fails, which has to do with the 'Tutorial page' routing. However, I don't really understand the error message or how to resolve this test failure.

**Comment by Drvanon** [comment]
The about page and the privacy policy are separate issues. It would be sensible to treat them in different pull requests.

**Comment by Drvanon** [comment]
Also, props for the file layout, I would have suggested that, but you were ahead of me.

**Comment by Yori-O92** [comment]
> 
> 
> The about page and the privacy policy are separate issues. It would be sensible to treat them in different pull requests.

A wow you are fast. Thanks for the feedback. Indeed, the privacy policy is more pertaining to Issue #26. There's also a pull request for that, since we just put the privacy policy as a document in the repository. I think I should stage-commit-push-pull-request more often for every issue separately in the future, but I'm not sure how to fix the mixup between issues now, since I've already committed and pushed everything onto the #18 branch...

**Comment by Drvanon** [comment]
This pull request was made obsolete by #85.

**Pull request closed without merging**